feelpp/tools/mesh/mesh_partitioner.cpp

```diff
@@ -47,7 +47,8 @@ int main( int argc, char** argv )
 	po::options_description meshpartoptions( "Mesh Partitioner options" );
 	meshpartoptions.add_options()
         ( "dim", po::value<int>()->default_value( 3 ), "mesh dimension" )
-        ( "shape", po::value<std::string>()->default_value( "simplex" ), "mesh dimension" )
+        ( "topdim-equal-realdim", po::value<bool>()->default_value( true ), "mesh dimension" )
```

Comment:
I think this is not the good way. @lberti could you replace it with an  option `realdim` (integer with no default value). And if not given, put realdim=dim and else just check that nRealDim >= nDim